src/ivoc/ivocmain.cpp

```diff
@@ -68,7 +68,7 @@ static PropertyData properties[] = {{"*gui", "sgimotif"},
                                     {"*brush_width", "0"},
                                     {"*double_buffered", "on"},
                                     {"*flat", "#aaaaaa"},
-#if defined(WIN32) || defined(CYGWIN)
+#if defined(MINGW)
```

Comment:
Sometimes we keep `WIN32`, sometimes `MINGW`.
Is there a rational behind it?

Reply:
There used to be a rationale. WIN32, MINGW, and CYGWIN are quite distinct in relation to the underlying system elements. At the moment with this PR, MINGW is the only supported  version on windows, so it makes some sense to have only that.  I don't know if that would be the case if there is ever an attempt to get this building with Visual C.  At any rate, with CYGWIN gone, it is now much clearer what gets compiled and what does not.

Reply:
Also there was some simplification with respect to DOS->Windows3->Windows and the getting rid of old ways of creating dlls.

Reply:
In some part we kept WIN32, is it normal?

Reply:
I did not review those in detail. If done at all, I'd prefer to do in another PR. E.g. in ivocmain.cpp I see
```
#if HAVE_IV
#ifdef WIN32
#include <IV-Win/MWlib.h>
void iv_display_scale(float);
#endif
```
Which  is definitely windows specific.

Reply:
Of course. so let's merge